frontend/components/ModelForm.tsx

```diff
@@ -402,6 +415,7 @@ export const ModelForm = (props: ModelFormProps) => {
           onDelete={onDelete}
           deleteVerb={deleteVerb}
           noun={noun}
+          requireOne={requireOne}
```

Comment:
It might make more sense to change this to `minimumRequired` or similar and make it an integer field, which makes it more flexible for future purposes.

Reply:
I don't want to just pass the number because this way I will need to implement counting the number of valid objects on the form, which is not ideal since different uses might have different validations, or I will have to pass the number of valid objects also from above. So I think it is easier to just pass whether it meets the required minimum or not. I can rename it to some more expressive.

Reply:
Ah, I see you've removed this altogether. That's even better!